Core/Graphics/Include/Shared/Babylon/Graphics/Device.h

```diff
@@ -28,6 +50,10 @@ namespace Babylon::Graphics
 
         // When enabled, back buffer will be premultiplied with alpha value.
         bool AlphaPremultiplied{};
+
+        // Format to use when creating the depth/stencil texture for the back buffer.
+        // Specify DepthStencilFormat::None to not create a depth/stencil texture.
+        DepthStencilFormat BackBufferDepthStencilFormat{DepthStencilFormat::Depth24Stencil8};
```

Comment:
So now the bgfx default will be D24S8, correct? What was it before (I think it was unspecified?)?